blog/_posts/2018-11-09-dvteam-farewell

```diff
@@ -1,51 +1,45 @@
 ---
-title: DV Team Farewell
 layout: post
 created_at: Fri Sep 7 2018
 permalink: blog/2018-11-09-dvteam-farewell
 current: blog
 author: Violeta & Diana
 twitter: artnerdnet
-categories: student-posts
+categories:
+- student-posts
 ---
 
-# Time's up!
```

Comment:
all of these markdown heading changes are unnecessary — could you please revert them all? 
i.e. we'd like to stick to headings with the `#` the way you had it in the original code :)

Reply:
done